frontend/translations/es.po

```diff
@@ -1229,8 +1229,11 @@ msgstr "Contraseña anterior"
 msgid "labels.only-yours"
 msgstr "Sólo los tuyos"
 
+msgid "labels.continue-with"
+msgstr "Continúa con"
+
 msgid "labels.or"
-msgstr "o"
+msgstr "o email"
```

Comment:
this can be changed. `labels.or` is clear, generic `or`, it can be changed to `or email`.